src/flows/recommendation_api/ff_new_tab_aggregated_engagement.py

```diff
@@ -0,0 +1,108 @@
+
+import pandas as pd
+from prefect import Flow, task, case
+from prefect.tasks.gcp.bigquery import BigQueryTask
+
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery
+from common_tasks.load_data import dataframe_to_feature_group
+from utils import config
+from utils.flow import get_flow_name, get_interval_schedule
+
+FLOW_NAME = get_flow_name(__file__)
+
+EXPORT_FIREFOX_TELEMETRY_SQL = """
+        WITH impressions_data AS (
+            SELECT s.*
+            FROM `moz-fx-data-shared-prod.activity_stream_live.impression_stats_v1` AS s
+            WHERE submission_timestamp > TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 1 DAY)
+            AND (source = 'TOP_STORIES' OR source = 'CARDGRID')
+            AND loaded IS NULL
+            AND normalized_country_code IS NOT NULL
+            AND array_length(tiles) >= 1
+        )
+        SELECT
+            CAST(flattened_tiles.id AS INT64) AS TILE_ID,
+            FORMAT_DATETIME("%Y-%m-%dT%H:%M:%SZ", CURRENT_DATETIME()) as UPDATED_AT,  -- Feature Store requires ISO 8601 time format
+            COUNT(*) AS TRAILING_1_DAY_IMPRESSIONS,
+            SUM(CASE WHEN click IS NOT NULL THEN 1 ELSE 0 END) AS TRAILING_1_DAY_OPENS,
+            -- For now, we only need 1 day trailing data, so leave the other ones at 0.  
+            0 AS TRAILING_7_DAY_IMPRESSIONS,
+            0 AS TRAILING_7_DAY_OPENS,
+            0 AS TRAILING_14_DAY_IMPRESSIONS,
+            0 AS TRAILING_14_DAY_OPENS,
+            0 AS TRAILING_21_DAY_IMPRESSIONS,
+            0 AS TRAILING_21_DAY_OPENS,
+            0 AS TRAILING_28_DAY_IMPRESSIONS,
+            0 AS TRAILING_28_DAY_OPENS
+        FROM impressions_data
+        CROSS JOIN UNNEST(impressions_data.tiles) AS flattened_tiles
+        GROUP BY TILE_ID
+        ORDER BY TRAILING_1_DAY_IMPRESSIONS DESC
+        LIMIT 16384 -- Limit to Snowflake's max list size. There are a lot of old items that still get some impressions.
+    """
+
+
+EXPORT_CORPUS_ITEM_KEYS_SQL = """
+    SELECT DISTINCT 
+        TILE_ID,
+        concat_ws(
+            -- corpus-engagement-v1 is keyed on the following three fields, separated by slashes.
+            '/', recommendation_surface_id, corpus_slate_configuration_id, corpus_item_id
+        ) as KEY,
+        RECOMMENDATION_SURFACE_ID,
+        CORPUS_SLATE_CONFIGURATION_ID,
+        CORPUS_ITEM_ID
+    FROM ANALYTICS.DBT_STAGING.STG_CORPUS_SLATE_RECOMMENDATIONS
+    WHERE TILE_ID in (%(tile_ids)s)
+"""
+
+
+@task()
+def df_column_to_list(df: pd.DataFrame, column_name: str):
+    return df[column_name].tolist()
+
+
+@task()
+def pd_merge(left: pd.DataFrame, right: pd.DataFrame) -> pd.DataFrame:
+    return pd.merge(left, right)
+
+
+@task()
+def df_drop(df: pd.DataFrame, columns: Union[str, List[str]]) -> pd.DataFrame:
+    return df.drop(columns=columns)
+
+
+@task()
+def df_is_empty(df: pd.DataFrame) -> bool:
+    return df.empty
+
+
+with Flow(FLOW_NAME, schedule=get_interval_schedule(minutes=15)) as flow:
```

Comment:
I set the schedule to 15 minutes somewhat arbitrarily. This is the same frequency that FeedMachine uses today.